modules/aws/go.mod

```diff
@@ -0,0 +1,32 @@
+module github.com/gruntwork-io/terratest/modules/k8s
+
+replace github.com/gruntwork-io/terratest/modules/logger => ../logger
+
+replace github.com/gruntwork-io/terratest/modules/random => ../random
+
+replace github.com/gruntwork-io/terratest/modules/retry => ../retry
+
+replace github.com/gruntwork-io/terratest/modules/customerrors => ../customerrors
+
+replace github.com/gruntwork-io/terratest/modules/files => ../files
+
+replace github.com/gruntwork-io/terratest/modules/ssh => ../ssh
+
+replace github.com/gruntwork-io/terratest/modules/collections => ../collections
```

Comment:
Does this work when the module is included remotely?

Reply:
I have no clue. It came to my attention it might represent a problem when importing the module, but in order to setup a test env, one would have to replace all occurrences of github.com/gruntwork-io with path/to/github/account. I have not done that.

So I figured I'll wait for the CI to go green and then do that test.